pkg/kexec/memory_linux.go

```diff
@@ -161,33 +161,33 @@ func (rs Ranges) Minus(r Range) Ranges {
 
 // FindSpace finds a continguous piece of sz points within Ranges and returns
 // the starting point.
-func (rs Ranges) FindSpace(sz uint) (start uintptr, err error) {
+func (rs Ranges) FindSpace(sz uint) (Range, error) {
```

Comment:
```suggestion
func (rs Ranges) FindRange(sz uint) (Range, error) {
```

Reply:
I think I'm gonna push back on this one -- `FindRange` sounds like we'd return an existing range from rs, instead of one tailored to sz.